openapi/apideck.accounting/types.bal

```diff
@@ -856,7 +856,7 @@ public type CreateCompanyResponse record {
 
 public type CustomField record {
     string id;
-    string|decimal|boolean|string[] value?;
+    string?|decimal?|boolean?|string[] value?;
```

Comment:
Seems there is an error in the openapi tool. Shall we manually change to 
```bal
string|decimal|boolean|string[]? value?;
```

Added an issue at openapi-tools https://github.com/ballerina-platform/openapi-tools/issues/810